website/content/en/docs/dev/Internals/_index.md

```diff
@@ -33,6 +33,7 @@ An instance directory contains the following files:
 
 Metadata:
 - `lima-version`: the Lima version used to create this instance
+- `lima-workdir`: the Lima workdir used to create this instance
```

Comment:
What is workdir? Why does it need to be a dir?

Reply:
The workdir is the $PWD from when the limactl start was first run.

It is used to find relative paths, for the scripts and the playbooks...

Otherwise, you need to `cd` to the directory before any `limactl`.

Or make all paths absolute, making them static and unportable?

Reply:
`File` can be extended to support relative paths?